### ISSUE TITLE
Can't open large files created by zip command

### ISSUE THREAD

**Issue by edtechd** [antelle/node-stream-zip#5]
Hello,

For some reason, I can't open large files created by Linux's zip command, I get an error "Invalid entry header". In the same time, it works with other software and passes zip's --test option.

Here is the link to the example 6.4 GB file  http://37.187.130.153:81/56eb37485f2aef9d7e00157d.zip
If you can't download this file, I can give some additional debug information.

Do you have any idea, is it possible to fix it?

**Comment by antelle** [comment]
Hello!
I get the following:
```
zip -T 56eb37485f2aef9d7e00157d.zip 
warning [56eb37485f2aef9d7e00157d.zip]:  2587553561 extra bytes at beginning or within zipfile
  (attempting to process anyway)
error [56eb37485f2aef9d7e00157d.zip]:  start of central directory not found;
  zipfile corrupt.
  (please check that you have transferred or created the zipfile in the
  appropriate BINARY mode and that you have compiled UnZip properly)
test of 56eb37485f2aef9d7e00157d.zip FAILED

zip error: Zip file invalid, could not spawn unzip, or wrong unzip (original files unmodified)
```

Could you please paste checksums for this file, to get sure it has been downloaded correctly?
Mine are:
MD5: bbb7a03158db84526bd28b0a2f826fee
SHA1: ff1160ade876af2f9f7ffa95afe2ad1bd410c18f

**Comment by antelle** [comment]
Fixed in v1.3.2

**Comment by edtechd** [comment]
Thank you! Checksum was correct.
